File: AppGenerateFiles/security/FrontModel/Tbl_Profile.js
//@ts-check
import { EntityClass } from "../../../WDevCore/WModules/EntityClass.js";
import { WAjaxTools, BasicStates } from "../../../WDevCore/WModules/WComponentsTools.js";
//@ts-ignore
import { ModelProperty } from "../../../WDevCore/WModules/CommonModel.js";
class Tbl_Profile extends EntityClass {
   /** @param {Partial<Tbl_Profile>} [props] */
   constructor(props) {
       super(props, 'EntitySecurity');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {Number}*/ Id_Perfil;
   /**@type {String}*/ Nombres;
   /**@type {String}*/ Apellidos;
   /**@type {Date}*/ FechaNac;
   /**@type {Number}*/ IdUser;
   /**@type {String}*/ Sexo;
   /**@type {String}*/ Foto;
   /**@type {String}*/ DNI;
   /**@type {String}*/ Correo_institucional;
   /**@type {Number}*/ Id_Pais_Origen;
   /**@type {String}*/ Estado;
}
export { Tbl_Profile }

File: CAPA_NEGOCIO/Gestion_Estudiantes/Model/Tbl_Profile.cs
using CAPA_DATOS;

namespace DataBaseModel
{public class Tbl_Profile : EntityClass
    {
        [PrimaryKey(Identity = true)]
        public int? Id_Perfil { get; set; }
        public string? Nombres { get { return $"{pariente.Primer_nombre} {pariente.Primer_nombre}" ; } }
        public string? Apellidos { get { return pariente.Nombre_completo; } }

        public DateTime? FechaNac { get; set; }
        public int? IdUser { get; set; }
        public string? Sexo { get; set; }
        public string? Foto { get; set; }
        public string? DNI { get; set; }
        public string? Correo_institucional { get; set; }
        public string? Estado { get; set; }
        public Parientes pariente { get; set; }

        public string GetNombreCompleto()
        {
            return $"{Nombres} {Apellidos}";
        }
    }
}